ios/nativeTemplate/Info.plist

```diff
@@ -30,6 +30,11 @@
 		<true/>
 		<key>NSExceptionDomains</key>
 		<dict>
+			<key>codepush.azurewebsites.net</key>
+			<dict>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
```

Comment:
Is this really necessary? Doesn't code push support https? I'd prefer to use a secure connection if possible.

Reply:
Confirmed and Done.

Reply:
Confirmed and done